src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.4
```

Comment:
Target branch is 6.3?

Reply:
branch 6.4 will be opened after 6.3 reaches it's first RC, for now I have no other branch to target